tools/proguard5.3.2/test/proguard/util/ArrayUtilTest.java

```diff
@@ -0,0 +1,614 @@
+package proguard.util;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+public class ArrayUtilTest {
+
+	@Test
+	public void testEqualByte() {
+		Assert.assertFalse(ArrayUtil.equal(
+				new byte[]{3, 2, 1}, new byte[]{1, 2, 3}, 4));
+
+		Assert.assertTrue(ArrayUtil.equal(new byte[]{1, 2, 3},
+				new byte[]{1, 2, 3}, 3)
+		);
+		Assert.assertTrue(ArrayUtil.equal(new byte[]{1, 2, 3},
+				new byte[]{1, 2, 3, 4}, 3)
+		);
```

Comment:
Maybe try with a null, blank array as well? I know you covered the method but for fun :)